src/apis/Kanga.js

```diff
@@ -0,0 +1,30 @@
+const { APIWrapper } = require('../')
+const snekfetch = require('snekfetch')
```

Comment:
SNEK  FETCH    A

Reply:
Oh we're using note-fetch now, aren't we?

Reply:
yes